complex/ft_comp.c

```diff
@@ -118,53 +118,40 @@ int ft_bind_comp(struct fid_ep *ep, uint64_t flags)
 	return 0;
 }
 
-int ft_comp_rx(void)
+int ft_comp_x(struct fid_cq *cq, struct ft_xcontrol *ft_x, char *x_str, int nowait)
```

Comment:
technically it should be 'static char *'.  The function doesn't change the string.